myapp/src/components/Navbar/MenuItems.js

```diff
@@ -0,0 +1,28 @@
+export const MenuItems = [
+    {
+        title:'Dartmouth',
+        url: "https://home.dartmouth.edu/", //where you put the link
+        cName: 'nav-links'
+    },
+    {
+        title:'Github',
+        url: 'https://github.com/MariaC27', //where you put the link
```

Comment:
nice! great idea for what to do with the navbar! 

an idea for the callback function assignment would be to link to different pages within your app using the navbar